### PR TITLE
Extend the documentation a bit

### PR DESCRIPTION
**Pull request by WhyNotHugo**
I've added some details which I think will be appreciated by future adopters.

### CODE REVIEW

File: docs/sbctl.8.txt [hunk 1]
```diff
@@ -113,12 +113,28 @@ EFI binary commands
...
 
 
+Bundles
+-------
+
+Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
+into a single file which can be easily signed. They also don't read parameters
+from external files, which makes them more tamper-proof.
+
+When a bundle is generated, its metadata is stored into the bundle database
```
Comment (WhyNotHugo) [inline docs/sbctl.8.txt:129]: Not sure if I prefer `metadata` or `manifest` here. Thoughts?

Comment (Foxboron) [reply docs/sbctl.8.txt:129]: `configuration` is better I think.

File: docs/sbctl.8.txt [hunk 2]
```diff
@@ -113,12 +113,28 @@ EFI binary commands
...
+        Removes a bundle from the list. This does not delete the bundle itself.
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
```
Comment (Foxboron) [inline docs/sbctl.8.txt:125]: `EFI executables` maybe? A little bit unsure if "binaries" convey the same meaning? I also think we should write `(initramfs, kernel and cmdline)`.

File: docs/sbctl.8.txt [hunk 3]
```diff
@@ -113,12 +113,28 @@ EFI binary commands
...
+into a single file which can be easily signed. They also don't read parameters
+from external files, which makes them more tamper-proof.
```
Comment (Foxboron) [inline docs/sbctl.8.txt:127]: Add a line about the security guarantees? `Normally only the kernel is signed with your secure boot keys, this leaves the kernel cmdline and initramfs unprotected.`

File: docs/sbctl.8.txt [hunk 4]
```diff
@@ -113,12 +113,28 @@ EFI binary commands
...
+
+When a bundle is generated, its metadata is stored into the bundle database
+(See **FILES**). Subsequent executions of *sbctl generate-bundles* will rebuild
+these bundles, so you don't need to re-specify all parameters after each system
+update.
+
+Hint: systemd-boot will automatically show entries for any bundles found in
+*$EFI/Linux/+++*+++.efi*.
```
Comment (Foxboron) [inline docs/sbctl.8.txt:135]: The proper way to write the path is `esp/EFI/Linux/+++*+++.efi` as we do on the wiki.

**Review by Foxboron — CHANGES_REQUESTED** [review]
Love the documentation changes. Thanks!

**Comment by Foxboron** [comment]
Please do look over if you can @ericonr :)

File: docs/sbctl.8.txt [hunk 5]
```diff
@@ -122,17 +122,20 @@ EFI binary commands
 Bundles
 -------
 
-Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
-into a single file which can be easily signed. They also don't read parameters
-from external files, which makes them more tamper-proof.
+Normally, only the kernel is signed with your secre boot keys. This leaves the
```
Comment (Foxboron) [inline docs/sbctl.8.txt:125]: "secure"

File: docs/sbctl.8.txt [hunk 6]
```diff
@@ -122,17 +122,20 @@ EFI binary commands
...
+files during boot makes the whole process more tamper-proof.
+
+When a bundle is generated, its configuration is stored into the bundle
+database (See **FILES**). Subsequent executions of *sbctl generate-bundles*
+will rebuild these bundles, so you don't need to re-specify all parameters
+after each system update.
 
 Hint: systemd-boot will automatically show entries for any bundles found in
-*$EFI/Linux/+++*+++.efi*.
+*esp/Linux/+++*+++.efi*.
```
Comment (Foxboron) [inline docs/sbctl.8.txt:138]: Still missing the `EFI` top-level directory :)

**Review by Foxboron — CHANGES_REQUESTED** [review]

File: docs/sbctl.8.txt [hunk 7]
```diff
@@ -113,12 +113,31 @@ EFI binary commands
...
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Normally, only the kernel is signed with your secure boot keys. This leaves the
+cmdline and initramfs unprotected.
```
Comment (ericonr) [inline docs/sbctl.8.txt:126]:
```suggestion
Normally, only the kernel is signed with your secure boot keys. This means the
kernel command line and initramfs can be changed without possibility of verification.
```

or something like that? Mainly, I'd like to have `kernel command line` spelled out.

File: docs/sbctl.8.txt [hunk 8]
```diff
@@ -113,12 +113,31 @@ EFI binary commands
...
+
+Bundles are EFI executables which pack all three (initramfs, kernel and
+cmdline) into a single file which can easily be signed. Avoiding any unsigned
```
Comment (ericonr) [inline docs/sbctl.8.txt:129]:
```suggestion
cmdline) into a single file which can be easily signed. Avoiding any unsigned
```

File: docs/sbctl.8.txt [hunk 9]
```diff
@@ -113,12 +113,31 @@ EFI binary commands
...
+files during boot makes the whole process more tamper-proof.
+
+When a bundle is generated, its configuration is stored into the bundle
+database (See **FILES**). Subsequent executions of *sbctl generate-bundles*
```
Comment (ericonr) [inline docs/sbctl.8.txt:133]:
```suggestion
database (see **FILES**). Subsequent executions of *sbctl generate-bundles*
```

**Comment by WhyNotHugo** [comment]
Feel free to squash when merging if you find the history is to noisy.

**Pull request merged**